frontend/src/view/Aplose_presentation/aplosestyles.css

```diff
@@ -0,0 +1,195 @@
+body, html {
+    margin: 0;
+    padding: 0;
+    width: 100%;
+    overflow-x: hidden;
+}
+
+/* Style principal pour la page APLOSE */
+#aplose-page {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    min-height: 100vh;
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+    width: 100%;
+}
+
+/* Style pour les bannières */
+.banner {
```

Comment:
For the same reason we don't write css for html or body tags, try adding `#aplose-page` selectors each time to be sure only the tags in your page get affected